Search.jsp

```diff
@@ -202,7 +209,7 @@ select * from comments where comment like '%<%=request.getAttribute("search2") %
 
 
 <footer>
-2020 copyright&copy;mjh.mohamed
+Copyright (C) 2019, 2020 [MIJC Team]

```

Comment:
@hema-001 

Thanks, MIJC Team。 Do you have a link to the detail of the team (e.g., its members)?

**Hui**

Reply:
For now!, NO.

But there will be an improvement to the code structure (i.e., refactoring) soon, I will include a link to the team detail that time.

IBRAHIM